src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -413,10 +413,21 @@ public static void MapInvalidateMeasure(IViewHandler handler, IView view, object
 		/// <param name="view">The associated <see cref="IView"/> instance.</param>
 		public static void MapContainerView(IViewHandler handler, IView view)
 		{
+			bool hasContainerOldValue = handler.HasContainer;
+
 			if (handler is ViewHandler viewHandler)
 				handler.HasContainer = viewHandler.NeedsContainer;
 			else
 				handler.HasContainer = view.NeedsContainer();
+
+			if(hasContainerOldValue != handler.HasContainer)
```

Comment:
I wonder if this should be a command? We can keep it "internal" for now, but just a command that is triggered when the container view changes? I think there are a few places that this is important to track:

Just looking at the Core layer in my PR: https://github.com/dotnet/maui/pull/13114/files#diff-e802e21561b0e99ffbb055b9ea43d833899fc9f06d91a64c5a9067c9427b7bcc there are several things I was adding to the `ContainerAffectingProperties`: `IView.Background`, `ILabel. VerticalTextAlignment`, `IView.Visibility`, `IView.Opacity`, `IView.Clip`, `IView.Shadow`, `IBorder.Border`, `IView.InputTransparent`.

I think what you have here is much better, no need to collect everything in a list, but if the command fires, each handler can control what it does when the container changes.

@PureWeen not sure if this is just abusing the system even more and we really should just do it right with a method? Maybe the view handler can get an internal `OnContainerViewChanged(PlatformView oldValue, PlatformView newValue)`?

All we need is a way for a handler to decide to re-fire mappers if the container changes. Commands are nice as they can be extended via third parties without having to derive from the existing handler. But, will anyone ever extend the mappers? Maybe we can have an internal virtual for now and think about it some more and maybe move to a command later?